documentation/docs/03-loading.md

```diff
@@ -93,6 +93,14 @@ If the page you're loading has an endpoint, the data returned from it is accessi
 
 `stuff` is passed from layouts to descendant layouts and pages, and can be filled with anything else you need to make available. For the root `__layout.svelte` component, it is equal to `{}`, but if that component's `load` function returns an object with a `stuff` property, it will be available to subsequent `load` functions.
 
+#### status
+
+`status` is `null` in normal circumstances, but will contain the error status code if there was a problem prior to the call to `load`.
+
+#### error
+
+`error` is `null` in normal circumstances, but will contain an error if there was a problem prior to the call to `load`.
+
```

Comment:
```suggestion
#### status

`status` is the HTTP status code when rendering an error page, or `null` otherwise.

#### error

`error` is the error that was thrown (or returned from a previous `load`) when rendering an error page, or `null` otherwise.

```